web_responsive/static/src/less/main.less

```diff
@@ -46,6 +46,24 @@ main {
 }
 
 .o_web_client {
+        &.o_fullscreen {
+        #oe_main_menu_navbar {
+            display: none;
+        }
+        .o_main .o_sub_menu {
+            display: none;
+         }
+        .navbar {
+            display: none;
+        }
+        .navbar-default main-nav{
+            display: none
+        }
+        .main-nav{
```

Comment:
Why not use multiple selectors once, like this:
```
#oe_main_menu_navbar,
.o_main .o_sub_menu,
.navbar,
.main-nav {
     display: none;
}
```

Reply:
Ohh, nice, haven't thought of it.
I'm not too familiar with less, I think of it like css maybe there lays the issue.

Reply:
Should `.navbar-default .main-nav` be removed?

Reply:
It did nothing before but we can keep it so the intention of the original developer is kept. :P

Reply:
Remove it please. We don't need uneeded code :wink:

Reply:
Done